micro-rdk-ffi/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+[package]
+name = "micro-rdk-ffi"
+authors.workspace = true
+description.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+version.workspace = true
+
+build = "build.rs"
+publish = false
+
+[lib]
+name = "micrordk"
+crate-type = ["staticlib"] # Creates static lib
+
+[target.'cfg(not(target_os = "espidf"))'.dependencies]
+env_logger.workspace = true
+local-ip-address.workspace = true
+micro-rdk = { workspace = true, features = ["data", "native", "provisioning"], default-features = true }
```

Comment:
we need to data service enabled by default if users end up wanting it in their idf project. We are most likely distribute the library through an idf component so it needs most features. But this is actually something we should discuss since here fore example we don't pull the default components in